autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionInfo.tsx

```diff
@@ -120,11 +120,24 @@ const SectionInfo: React.FC<SectionInfoProps> = ({
     </Typography>
   );
 
+  // adds a tooltip for meeting types that aren't very obvious ex -> INS, PRL, etc.
+  const formatMeetingType = (mtg: Meeting): JSX.Element => {
+    const meetingTypeDescription = MeetingTypeDescription.get(mtg.meetingType);
+    if (meetingTypeDescription != null) {
+      return (
+        <Tooltip title={meetingTypeDescription} arrow placement="bottom" PopperProps={{ disablePortal: true }}>
+          <span className={styles.meetingType}>{MeetingType[mtg.meetingType]}</span>
+        </Tooltip>
+      );
+    }
+    return <span>{MeetingType[mtg.meetingType]}</span>;
```

Comment:
honestly, you could probably return a string

Reply:
but what about the function return type being a JSX element?

Reply:
If it's a string, line 140 will end up just being `<td>{"string here"}</td>` which is valid